### DISCUSSION TITLE
Query Word Wrap on Label

### DISCUSSION THREAD

**Discussion by rsrinivasanhome** [simonec73/threatsmanager#33] [General]
@simonec73  is there any way to word wrap the labels associated to a flow ?

Is there any way to add a label to the threat model so that we can add some explanation in the diagram.

**Comment by simonec73** [comment]
Thank you, rsrinivasanhome.

The first answer is that no, you cannot word wrap labels. That would not work well on all mediums (I'm thinking of Word reports, for example). Therefore, it is not something I'm planning to add anytime soon.
The recommendation is to use short text. You can use the Description field to include your extended explanation of what the flow does.

Analogously, I'm not planning to add text stencils like those in TMT. This is because the said text fields would be unrelated to anything else, and therefore it may be difficult to understand it if extrapolated from the diagram. As a result, it would not be possible to include the said text stencils in a report of any type. Given that the text stencils must be seen in the diagram they are part of to be understood, it may not be possible to document them as part of the Word report, particularly if the diagram is big enough. Therefore, you would need to give away the Threat Model itself which sometimes is not feasible.
The recommendation is to use the Description field that you can find on most objects. You can also use Annotations (see [Annotations - Threats Manager Studio](https://threatsmanager.com/training/advanced/extensions/annotations/)). The text you include in the Notes can also be included in the Word report.

I hope it helps.

Simone

**Comment by simonec73** [comment]
A recent release has implemented Word wrap for labels (2.1.5).
